README.md

```diff
@@ -4,23 +4,40 @@ This README outlines the details of collaborating on this Ember addon.
 
 ## Installation
 
-* `git clone <repository-url>` this repository
-* `cd @ember/optional-features`
-* `yarn install`
+```bash
+ember install @ember/optional-features
+```
 
-## Running
+## Usage
 
-* `ember serve`
-* Visit your app at [http://localhost:4200](http://localhost:4200).
+### List available features
+Features will only be available in canary builds of ember-source. To list all available features run:
```

Comment:
> in canary builds of ember-source

This is not actually true. It is intended to work with stable builds (unlike experimental feature flags, this is to opt-in/out of things that we can't easily "just change" due to compatibility/semver concerns). However, as of now, the earliest version that has any optional features is 3.1, which is still in its canary cycle atm.

Reply:
<img width="1277" alt="slack_-_emberjs_community" src="https://user-images.githubusercontent.com/55829/35519237-835b0a7e-04c8-11e8-8baf-6e9df441892d.png">